ONSdigital/eq-author
# Publish to the survey registry

**What**
Once this change is commit, we can change the author application to publish questionnaires to the survey registry.

https://github.com/ONSdigital/eq-survey-registry/pull/1

**How to test**
1. `docker-compose build`
2. `docker-compose up`
3. Create a survey and questionnaire on the author application
4. Publish the questionnaire
5. Using the survey runner view the live questionnaire

Note: in the logs you should see that the request to get the questionnaire schema goes to the survey registry

**Who can review**
Anyone apart from @warren-methods
